tfac/figures/figure2.py

```diff
@@ -1,42 +0,0 @@
-"""
-This creates Figure 2 - protein factors.
```

Comment:
Why delete this? Aren't we going to need to plot the factors?

Reply:
will bring it back in a sec.